software/tests/baremetal/Makefile

```diff
@@ -26,7 +24,7 @@ all: $(TESTS)
 $(TESTS): % : $(BIN_DIR)/% $(TESTS_DIR)/Makefile $(shell find $(RUNTIME_DIR)/**.{S,c,h,ld} -type f)
```

Comment:
Same here, it would be triggered to generate all of the data even not be used in the target kernel. But low-priority to change, it's already good enough for me.

Reply:
This was fixed.